Licensee/parse-pdq-dtd.py

```diff
@@ -34,25 +34,29 @@
     "PoliticalSubUnit",
     "Protocol",
     "Summary",
-    "Term"
-])
-DROP = set(["Protocol"])
+    "Term",
+}
+DROP = {"Protocol"}
 DROP = set()
 KEEP = DOCTYPES - DROP
```

Comment:
This looks odd to me.  I'm assuming you're trying to subtract the dropped doctypes from the list of all doctypes.

Reply:
As I recall, the first `DROP` included `Protocol` in a previous run because the "old" version of the DTD had `Protocol` as a document type and we were verifying that the "new" version had successfully dropped it. Now that the `Protocol` document is really gone (so it won't be present in either the "old" or the "new" DTD being compared) it's removed from the `DROP` set. You might wonder why `Protocol` is still in the `DOCTYPES` set. It doesn't hurt anything to be there, but it's presence is no longer useful, so we could remove it next time around. I'm assuming that it's the composition of the sets that puzzled you, and not the new Python 3 syntax for set constants (though it's the syntax which change for Kepler, not the composition of the sets).

Reply:
I was confused about two thing:
a) I was not aware that you could create a set using curly braces.  I thought you'd do that with parenthesis 
b) Secondly I was confused why you would create an empty set "DROP" and therefore preventing the "Protocol" doctype to be removed.

Reply:
I hope my earlier comment took care of (b). For (a), see https://docs.python.org/3.0/whatsnew/3.0.html#new-syntax

> Set literals, e.g. {1, 2}. Note that {} is an empty dictionary; use set() for an empty set. Set comprehensions are also supported; e.g., {x for x in stuff} means the same thing as set(stuff) but is more flexible.